gulpfile.js

```diff
@@ -10,10 +10,10 @@ gulp.task('sass', function () {
         .pipe(browserSync.stream());
 });
 
-gulp.task('watch', ['sass', 'html', 'browser-sync'], function () {
+gulp.task('watch', ['sass', 'html', 'static', 'browser-sync'], function () {
     gulp.watch(['./src/**/*.scss'], ['sass']);
     gulp.watch(['./src/*.html'], ['html']);
-
+    gulp.watch(['./src/img/**'], ['static']);
```

Comment:
Schaut einfach, ob sich eine Datei in dem Ordner geändert hat. Leider erkennt es dzt. nicht automatisch, ob **neue Dateien** hinzugekommen sind. In diesem Fall musst du also `gulp watch` neustarten, oder manuell in einem zweiten Terminal `node_modules/.bin/gulp static` ausführen. Dieser Befehl führt einfach nur den unten als `static` definierten Task aus und kopiert die Dateien.